components.css

```diff
@@ -0,0 +1,252 @@
+.aside {
+  flex-basis: 50%;
+}
+.main-content {
+  margin: 4.5rem 9rem 1.5rem;
+  gap: 4rem;
+}
+.heading-1 {
+  font-size: 3rem;
+  margin-bottom: 1rem;
+}
+.heading-2 {
+  margin-bottom: 1rem;
+}
+.heading-3 {
+  font-size: 1.75rem;
+  margin-top: 1rem;
+}
+.heading-4 {
+  font-size: 1.3rem;
+  font-weight: 600;
+  margin: 2rem 0;
+}
+.container-box {
+  border: 1px solid var(--grey-2);
+  padding: 1rem 2rem;
+  margin: 2rem 0;
+  justify-content: space-evenly;
+  min-height: 10vh;
+}
+
+.icon-1 {
+  color: var(--purple);
+  min-width: 4rem;
+  min-height: 4rem;
+  display: flex !important;
+}
+.usage-list {
+  margin-bottom: 3rem;
+}
+
+/* *******************alert******************** */
+.alert {
+  padding: 1rem;
+  margin-bottom: 1rem;
+  border: 1px solid;
+}
+.alert-primary {
+  color: #0746a3;
+  background-color: #ccdffd;
+  border-color: #b7d3fb;
+}
+.alert-danger {
+  color: #861d26;
+  background-color: #fcd2d6;
+  border-color: #f5bdc2;
+}
+.alert-success {
+  color: #0f5132;
+  background-color: #d1e7dd;
+  border-color: #badbcc;
+}
+.alert-warning {
+  color: #644b00;
+  background-color: #ffeaa6;
+  border-color: #fbe7ab;
+}
+.alert-dark-mode {
+  color: var(--white);
+  background-color: var(--black);
+  border-color: var(--grey);
+}
+
+/* *******************badges****************** */
+.badge {
+  padding: 0.8rem;
+  position: relative;
+}
+.notify-count {
+  font-size: 1.4rem;
+  max-width: 1.6em;
+  max-height: 1.6em;
+  min-width: 1.6em;
+  min-height: 1.6em;
+  position: absolute;
+  top: 0;
+  color: var(--white);
+}
+.notify-count-right {
+  background-color: var(--sea-green);
+  right: 0;
+}
+.notify-count-left {
+  background-color: var(--red);
+  left: 0;
+}
+.badge-icon {
+  font-size: 3.5rem !important;
+}
+.notify {
+  max-width: 1em;
+  max-height: 1em;
+  min-width: 1em;
+  min-height: 1em;
```

Comment:
why are you using min and max-width the same?

Reply:
Using width will simply specify fixed width over the element without paying attention to its content (so it can overflow)
So if I specify min-width and max-width, I will set up a lower and upper bound and if both are equal it will be the same as simply specifying a width.